core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -94,6 +96,10 @@ class MultiCloudWorkspaceService(userInfo: UserInfo,
           parentSpan)
       }, TransactionIsolation.ReadCommitted)
       )
+      _ = logger.info(s"Enabling leonardo app in WSM [workspaceId = ${workspaceId}]")
+      _ <- traceWithParent("enableLeoInWSM", parentSpan)(_ =>
+        Future(workspaceManagerDAO.enableApplication(workspaceId, wsmConfig.leonardoWsmApplcationId, userInfo.accessToken))
```

Comment:
Do we need to worry about this throwing an exception (with respect to the span)?

Reply:
Do we know the typical time taken to enable an application? Wondering if/how this operation will impact createWorkspace's performance.